Save text file

citlaligm opened the pull request:
## Overview
Add a button to parse and save IMU, GPS, or RPM logs to a text file if desired and fixes a small bug.

## Changes
- Adds a button to optionally save the data for IMU, GPS, RPM to a text file. If the button is set to true, you will be prompted where to save the file.
![image](https://user-images.githubusercontent.com/10260895/99313820-d22d0980-2825-11eb-8396-636aafb41e69.png)

- Once we read the complete file, the file is giving an error because it has reached the end. We needed to clear this "expected" error to avoid confusion. 

## Test
- Log data using the `mono_logger.vi` for IMU, GPS, RPM, set to True the `Save text file?` button. The file should contain a line for each sample on the binary file.

bradleynull reviewed (changes requested):
@citlaligm This is working for me with IMU, RPM, and GPS to save to text files. A couple of comments:

* The text for the window that appears when running the parser is confusing and the file filter is set to "All files (*.*)". Can we set the file filter to ".txt" and change the text to indicate that this is the window to select the text file location: 
![image](https://user-images.githubusercontent.com/6026642/99419079-52ed1380-28c1-11eb-876e-9274cc5d98f9.png)

* When trying to select the directory for "Save images to..." the file filter is set to "All files (*.*)" and will not let me set a directory. Copy+pasting a directory into the box (instead of using the GUI selection) lets me properly set it and save images.
![image](https://user-images.githubusercontent.com/6026642/99419211-787a1d00-28c1-11eb-9159-db40bbb65e68.png)

* This can be saved for later, but I would suggest a feature that lets you auto-populate the sensor type drop down would be helpful (automatically generate the pink boxes): 
![image](https://user-images.githubusercontent.com/6026642/99419321-99db0900-28c1-11eb-82da-49e47e3f67a2.png)

citlaligm commented:
@bradleynull Thanks for the feedback.

1. Unfortunately, is not possible to set a filter on this pop-up, the alternative would be to create another path control similar to the "save images to...", I was trying to avoid this to reduce confusion on what boxes to fill. However, I modified the header of the window to give more info to the user. Let me know what do you think.

2. Fixed. Good catch. Thanks.

3. I think, it is a great idea to explore. Saving for a future PR.

bradleynull reviewed (approved):
@citlaligm Looks good to go now. Approved!
